src/pages/Movie/TopInfo/ToggleToFavorite/__tests__/containet.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import configureStore from 'redux-mock-store';
+import { shallow } from 'enzyme';
+
+import ConnectedToggleToFavorite from '../container';
+import { toggleToFavorite } from '../../../../../../state/favorites/actions';
+
+jest.mock('../../../../../../state/favorites/actions', () => ({
+  toggleToFavorite: jest.fn()
+}));
+
+describe('ToggleToFavorite container', () => {
+  const store = configureStore()({});
+  const props = {
+    store,
+    id: 1,
+    favorite: false
+  };
+  store.dispatch = jest.fn();
+  const wrapper = shallow(<ConnectedToggleToFavorite {...props} />); /* eslint-disable-line */
+  const container = wrapper.dive();
+  const instance = container.instance();
+
+  it('matches snapshot', () => {
+    expect(container).toMatchSnapshot();
+  });
+
+  it('changeFavoriteStatus() trigger toggleToFavorite()', () => {
+    const {
+      id,
```

Comment:
You already have this in `props` variable

Reply:
Fixed